### PR TITLE
Lighting de-lagging

### LINKED ISSUE #216: Fix shadows
Shadows have been turned off / removed until we can figure out what their problem is.

### PR DESCRIPTION
**Pull request by d34d10cc**
The client no more has extra lag when running with lights enabled, so I reenabled them, only the 'normal' amount of lag from the delta thing.

Fixes #216.

Please no more Texture.CopyToImage(), thanks.

### CODE REVIEW

File: SS14.Client/State/States/GameScreen.cs.rej [hunk 1]
```diff
@@ -0,0 +1,10 @@
+diff a/SS14.Client/State/States/GameScreen.cs b/SS14.Client/State/States/GameScreen.cs	(rejected hunks)
```
Comment (d34d10cc) [inline SS14.Client/State/States/GameScreen.cs.rej]: what

**Review by psykzz — CHANGES_REQUESTED** [review]
Any chance we can move to using a class to abstract away your huge messy dictionary?

We can still use the string name as key but instead of saving an opacity map and an image, just store a class that handles all that for us.

This is almost impossible to maintain as it is.

**Comment by PJB3005** [comment]
Yeah this is getting ridiculous.

**Review by psykzz — APPROVED** [review]

**Review by ZoldorfTheWizard — APPROVED** [review]
Its another hacky fix but it does fix the performance loss caused by reading the texture then immediately writing the changes

**Pull request merged**